### ISSUE TITLE
UI and API for selecting color profile is unnecessarily limiting.

### ISSUE THREAD

**Issue by pixelzoom** [phetsims/joist#836]
In the Preferences dialog, the UI for selecting a color profile looks like this:

<img width="774" alt="screenshot_1803" src="https://user-images.githubusercontent.com/3046552/185973158-11af5961-8123-4309-8cc9-4f3a5c8e37c1.png">

And the API looks like this (example from gravity-and-orbits-main.ts):

```typescript
  const simOptions: SimOptions = {
...
    preferencesConfiguration: new PreferencesConfiguration( {
      visualOptions: {
        supportsProjectorMode: true
      }
    } )
  };
```

This assumes that there are only 2 color profiles: 'default' and 'projector'.  PhET specifically designed the color-profile API to support additional profiles, to meet future needs.  While no sim currently has more than 2 profiles, this UI and API will be insufficient the first time we need a 3rd profile.

A more flexible API would pass the color profiles to the sim (along with their descriptions), and the UI would present those profiles as radio buttons.

**Comment by arouinfar** [comment]
> This assumes that there are only 2 color profiles: 'default' and 'projector'. PhET specifically designed the color-profile API to support additional profiles, to meet future needs. While no sim currently has more than 2 profiles, this UI and API will be insufficient the first time we need a 3rd profile.

This came up in design meeting  while we considered UI options for Projector Mode, but decided to use a switch to match some of the other controls already in the Preference Dialog, like Interactive Highlights or the Voicing/Sound switches. The previous Projector Mode checkbox was also a binary control.  Since there are no plans to add support for additional color profiles for the foreseeable future, this approach seemed reasonable. 

Radio buttons would be more generalizable. If we want to consider it, here are some mockups including a fictitious 3rd color profile. 
<img width="930" alt="image" src="https://user-images.githubusercontent.com/8419308/186005604-b9f77011-2382-4dc4-98f2-441ba4079d8c.png">

From an aesthetic perspective, I prefer the current implementation with the switch. It's clear and easier to understand, but I also recognize that it's not future-proof.

**Comment by pixelzoom** [comment]
I get it.  And I'd prefer the cleanness of the on/off switch -- if it was really a binary choice.

But it's not binary. So there's a big mismatch between the UI and color-profile API here. Are we certain that there will be no need to add additional color profiles? Nothing related to a11y or research, like a high-contrast profile?

**Comment by zepumph** [comment]
Can we make the argument though that it is at least not more of a mismatch as what is already implementing ProjectorMode checkbox (which uses assumptions and assertions to map colorProfile into projector mode and "other")? We can always change the UI when it is time to add more color profiles.

For context, you are catching me after ~5 months of scope creep on the upcoming PhET-iO batch release, so anything we can do to move towards publication is a win to me!

**Comment by pixelzoom** [comment]
Agreed that it is no worse than using ProjectorModeCheckbox in an Options dialog.

**Comment by jessegreenberg** [comment]
> The previous Projector Mode checkbox was also a binary control. Since there are no plans to add support for additional color profiles for the foreseeable future, this approach seemed reasonable.

Agreed and this is what we discussed in the meeting. But other sim code/tools do support multiple color profiles, it makes sense for this dialog to support that. If we need multiple profiles later will that require PhET-iO migration rules (I am not really sure what is involved for that)? Code side, it should be easy to change the UI/API for multiple profiles. ComboBox might also work and conserve space. Now could be a good time while work in Preferences is fresh.

@arouinfar @kathy-phet can you comment on what is best for design here and if the UI needs to be ready for multiple profiles now (I think mostly for PhET-iO compatibility in the future)?

**Comment by kathy-phet** [comment]
I would like to support a forward planning version, where we go with radio buttons, so the choices are all visible to the user. 

I prefer "Display Mode" or "Display Options" or something with the word "Display", rather than "Color Profile".

**Comment by arouinfar** [comment]
Here's a mockup that I reviewed with @kathy-phet. It contains all of the options we can think of that would fall under "Visual", including the BAM-specific high contrast particles.

<img width="843" alt="image" src="https://user-images.githubusercontent.com/8419308/187552838-e4a25da0-083f-446a-b072-ab026bbb68e4.png">

**Comment by zepumph** [comment]
Discussed during PhET-iO meeting today. @arouinfar and @kathy-phet would like a "Display Mode" radio button group. 

* Please use the above mockup exactly except that "Default" doesn't need any extra description. 
* Keep `colorProfileProperty` as the name of the mode Property, and don't change the name of the linkedElement/references key in the PreferencesModel.visualModel.

**Comment by samreid** [comment]
In discussion with @arouinfar and @kathy-phet, we agreed this is non-blocking and can actually be deferred until we have a 3rd color profile.  We don't know what sim that would be at the moment.

**Comment by samreid** [comment]
When this issue undefers, it would also be a good time to work on https://github.com/phetsims/joist/issues/865
